lib/app/app.rb

```diff
@@ -19,7 +19,10 @@
 require 'connectors_shared'
 require 'connectors_app/config'
 require 'connectors_sdk/base/registry'
-require 'connectors_async'
```

Comment:
I've reverted the change back, but tbh I like specific requires instead of full namespace ones, just for sake of readability and ease of understanding